README.md

```diff
@@ -0,0 +1,29 @@
+# Test Generators
```

Comment:
Could you also drop in a `.tool-versions` file for asdf and a `.node-version` for nvm users?

https://github.com/LibertyDSNP/sdk-ts/blob/main/.tool-versions

Reply:
done.